src/rules/sortImportsRule.ts

```diff
@@ -0,0 +1,242 @@
+'use strict';
```

Comment:
Interesting - I think it's in the new rule template (and I add it reflexively anyways :) ) Anyhow, I'll remove it.